workflow-dev/tpl/deis-objectstorage-secret.yaml

```diff
@@ -7,23 +7,23 @@ metadata:
   labels:
     heritage: deis
   annotations:
-    deis.io/objectstorage: "{{.storage}}"
+    deis.io/objectstorage: "{{ or (env "STORAGE_TYPE") (.storage)}}"
 type: Opaque
-data: {{ if eq .storage "minio" }}
-  accesskey: OFRaUlkySlJXTVBUNlVNWFI2STU=
-  secretkey: Z2JzdHJPdm90TU1jZzJzTWZHVWhBNWE2RXQvRUk1QUx0SUhzb2JZaw=={{ else if eq .storage "gcs" }}
-  key.json: {{ b64enc .gcs.key_json }}
-  registry-bucket: {{ b64enc .gcs.registry_bucket }}
-  database-bucket: {{ b64enc .gcs.database_bucket }}
-  builder-bucket: {{ b64enc .gcs.builder_bucket }}{{ else if eq .storage "azure" }}
-  accountname: {{ b64enc .azure.accountname }}
-  accountkey: {{ b64enc .azure.accountkey }}
-  registry-container: {{ b64enc .azure.registry_container }}
-  database-container: {{ b64enc .azure.database_container }}
-  builder-container: {{ b64enc .azure.builder_container }}{{ else if eq .storage "s3" }}
-  accesskey: {{ b64enc .s3.accesskey }}
-  secretkey: {{ b64enc .s3.secretkey }}
-  region: {{ b64enc .s3.region }}
-  registry-bucket: {{ b64enc .s3.registry_bucket }}
-  database-bucket: {{ b64enc .s3.database_bucket }}
-  builder-bucket: {{ b64enc .s3.builder_bucket }}{{ end }}
+data: {{ if or (eq (env "STORAGE_TYPE") "gcs") (eq .storage "gcs")}}
+key.json: {{ or (env "GCS_KEY_JSON") (.gcs.key_json) | b64enc}}
```

Comment:
there's a lot going on here and it looks the previous indentation and spacing got removed -- any way we can add back some indentation/spacing for readability?